docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
```

Comment:
What's the docker-compose file for?

Reply:
That's for local deployment.
Its building the docker image and runs it on a computer.
I saw Dima is using it (as mentioned in the other issue) so I said it's a good idea to have it here too.

Reply:
It's more of a user preference. We can add a section to the documentation site. We don't need it in the repo's root.

Reply:
From my experience, it's common to leave docker-compose file on the git as a file not as a doc.
Anyway, this is not harmful at all.

Reply:
I think we can drop it. Feels unrelated.

Reply:
@YogevBokobza, let's remove the docker-compose file and merge. The arm64 is essential enough for a quick release.